src/SpinnerKernel1.java

```diff
@@ -0,0 +1,51 @@
+import components.standard.Standard;
+
+//I don't know why my Standard isn't being imported :(
+public interface SpinnerKernel1<Slice>
```

Comment:
The naming conventions are backward. This should be SpinnerKernel and the file above should be SpinnerKernel1.